### DISCUSSION TITLE
How to use LoRA correctly?

### DISCUSSION THREAD

**Discussion by Vi3344** [Acly/krita-ai-diffusion#648] [Q&A]
Hi, I need to change the rendering style of my image. I'm used to using LoRA extensions on A1111, but with Krita, it doesn't work. There's no change in the rendering. Is there anything specific I should do for it to work?
Thanks

**Comment by Muskelmagier** [comment]
Rendering style? I am a little confused what you mean by that. 

Do you mean the Scheduler? like "eula", "Eula ancestral", "DDIM" and so on 

Or do you mean how to use a style lora? because for style lora you simply create in the options a new style and then Add the Lora there

**Comment by Vi3344** [comment]
Ok, So it's as simple as I thought. I don't know why but it doesn't work with me. I add the lora in the option and nothing change.
I probably need to find a UI manager for comfyUI.
Thanks for your answer
